packages/ui/src/i18n/dictionaries/authenticator/es.ts

```diff
@@ -14,10 +15,12 @@ export const esDict = {
   'Confirmation Code': 'Código de confirmación',
   Confirming: 'Confirmando',
   'Create a new account': 'Crear una cuenta nueva',
+  'Create Account': 'Crear cuenta',
   'Creating Account': 'Creando cuenta',
   Email: 'Email',
+  'Enter your email': 'Email',
```

Comment:
Shouldn't this be something like "Escriba su Email"?

Reply:
Yes, let me fix it